ingestion/src/metadata/ingestion/ometa/mixins/table_mixin.py

```diff
@@ -101,7 +102,7 @@ def ingest_table_queries_data(
         for query in table_queries:
             self.client.put(
                 f"{self.get_suffix(Table)}/{table.id.__root__}/tableQuery",
-                data=query.json(),
+                data=json.dumps(query),
```

Comment:
please, check that we are passing the proper model `SqlQuery`. If we are sending the right type we should be able to run `query.json()`.

Reply:
so if possible, revert this change and fix how we send the sample data